packages/core-react/index.d.ts

```diff
@@ -8,4 +8,16 @@ export type TextFieldProps = {
 
 declare const TextField: React.FC<TextFieldProps>;
 
-export { TextField };
+export type ButtonProps = {
+  Tag?: 'button' | 'a';
+  children?: React.ReactNode;
+  type?: 'primary' | 'secondary' | 'tertiary';
+  compact?: boolean;
+  danger?: boolean;
+  disabled?: boolean;
+  className?: string;
+};
```

Comment:
I think we can add  `& React.HTMLProps<HTMLButtonElement>` at the end here too. (I'll add and push!)